roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -41,12 +44,22 @@ class RooFuncWrapper final : public RooAbsReal {
 protected:
    double evaluate() const override;
 
+   void buildCode(RooAbsReal const &head, std::string &code, RooArgSet const & /* paramSet */, RooArgSet const &ObsSet,
+                  const RooAbsData *data);
+
 private:
    void updateGradientVarBuffer() const;
 
    using Func = double (*)(double *, double *);
    using Grad = void (*)(double *, double *, double *);
 
+   void
+   loadParamsAndObs(std::string funcName, RooArgSet const &paramSet, RooArgSet const &ObsSet, const RooAbsData *data);
```

Comment:
```suggestion
   loadParamsAndObs(std::string funcName, RooArgSet const &paramSet, RooArgSet const &obsSet, const RooAbsData *data);
```
In RooFit, variable names start lower case (also for `ObsSet` -> `obsSet` in the cxx file).

Reply:
Yeah I know that, it's probably a typo.